events/README.md

```diff
@@ -0,0 +1,36 @@
+# Events
+
+```
+{
+  'name': string,
+  'description': string,
+  'room_id':string,
+  'template_id': string,
+  'enabled': boolean,
+  'lock': boolean,
+  'region': <'in' | 'us' | 'eu' | 'auto'>,
+  'peers': [
+    "<peer_id>",
+    "<peer_id>",
+    "<peer_id>"
+   ]
+  'created_by': {
+    ref: 'User',
```

Comment:
It can also be similar as peer. 
We can use whole object of the user who created this.